src/engine/sparqlExpressions/SparqlExpressionTypes.h

```diff
@@ -6,15 +6,16 @@
 #ifndef QLEVER_SPARQLEXPRESSIONTYPES_H
 #define QLEVER_SPARQLEXPRESSIONTYPES_H
 
-#include "../../global/Id.h"
-#include "../../util/AllocatorWithLimit.h"
-#include "../../util/ConstexprSmallString.h"
-#include "../../util/Generator.h"
-#include "../../util/HashMap.h"
-#include "../../util/TypeTraits.h"
-#include "../QueryExecutionContext.h"
-#include "../ResultTable.h"
 #include "SetOfIntervals.h"
+#include "engine/QueryExecutionContext.h"
```

Comment:
The `SetOfIntervals.h` should also be `engine/sparqlExpressions/SetOfIntervals.h` (if that is the right path).